CMSgov/easi-app
# Easi 3473/transaction helper functions

# EASI-3473

## Changes and Description

- Port Transaction Helper functions implemented in the MINT Application (See [PR 820](https://github.com/CMSgov/mint-app/pull/820))
     - With Transaction
     - Named Preparer
- Converted the New TRB Request functionality to use the With Transaction block
      - removed the transaction implementation from the store New TRB request store method
      - extracted new TRB request form to it's own store method
      - TRB request, form and attendee are all part of a transaction now.
- WithTransaction unit test implementation


## How to test this change
1. Bring up the application
2. Create a new TRB request
3. Verify that functionality continues to work as expected

Also, verify unit tests look as expected and function appropriately.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
